class.go

```diff
@@ -21,9 +21,11 @@ type Class struct {
 	UniqueQueries uint     // unique number of queries in class
 	Example       *Example `json:",omitempty"` // sample query with max Query_time
 	// --
-	outliers uint64
-	lastDb   string
-	sample   bool
+	outliers                uint64
+	lastDb                  string
+	sample                  bool
+	maxQueryTime            float64
+	MaxQueryCommentMetadata map[string]string
```

Comment:
Let's call this just `Metadata`. The max query time stuff is something we do internally.